run.sh

```diff
@@ -0,0 +1 @@
+./jdk-17.0.2.jdk/Contents/Home/bin/java -XstartOnFirstThread -jar Tanks.jar mac
```

Comment:
Do we want to keep this script in the pull request? It looks like something used for testing.

Reply:
Good point, however, we do want a way for non-savy users to run the game from the build. As such, I have modified the run command to work more universally on *nix systems.